modules/exploits/windows/ftp/pcman_put.rb

```diff
@@ -0,0 +1,80 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+
+class Metasploit3 < Msf::Exploit::Remote
+  Rank = NormalRanking
+
+  include Msf::Exploit::Remote::Ftp
+
+  def initialize(info = {})
+    super(update_info(info,
+      'Name'           => 'PCMAN FTP Server v2.0.7 Buffer Overflow - PUT Command',
+      'Description'    => %q{
+          This module exploits a buffer overflow vulnerability found in the PUT command of the
+          PCMAN FTP v2.0.7 Server. This requires authentication but by default anonymous
+          credientials are enabled.
+      },
+      'Author'         =>
+          [
+            'Jay Turla @shipcod3',      # Initial Discovery
+            'Chris Higgins @ch1gg1ns'   # msf Module
+          ],
+      'License'        => MSF_LICENSE,
+      'References'     =>
+        [
+          [ 'EDB',   '37731']
+        ],
+      'DefaultOptions' =>
+        {
+          'EXITFUNC' => 'process',
+          'VERBOSE'  => true
```

Comment:
Copied from an existing module as a template, though yeah I did think it was odd. I can take it out.